packages/capture/README.md

```diff
@@ -0,0 +1,19 @@
+<p align="center">
+  <a href="https://argos-ci.com/?utm_source=github&utm_medium=logo" target="_blank">
+    <img src="https://raw.githubusercontent.com/argos-ci/argos/main/resources/logos/logo-github-readme.png" alt="Argos" width="300" height="61">
```

Comment:
Why do you use an other logo for some packages ? It look like we have a legacy logo not updated with Argos visual chart.

Reply:
It is the same logo for all packages. Where is it different in others?

Reply:
This "package logo" is not the same as website logo and has different graphic chart

Reply:
Yeah to support dark mode on GitHub ;)